src/ahrens_lab_to_nwb/yu_mu_cell_2019/yu_mu_cell_2019_metadata.yml

```diff
@@ -1,11 +1,69 @@
   institution: Janelia Research Campus
-  lab: Misha Ahrens
+  lab: Ahrens Lab
   experimenter:
     - Mu, Yu
-    - Ahrens, Misha
+    - Bennett, Davis V.
+    - Rubinov, Mikail
+    - Narayan, Sujatha
+    - Yang, Chao-Tsung
+    - Tanimoto, Masashi
+    - Mensh, Brett D.
+    - Looger, Loren L.
+    - Ahrens, Misha B.
 Subject:
   species: Danio rerio
+
+Ophys:
+  Device:
+    - name: Microscope
+      description: Confocal imaging was done using a Zeiss LSM 710, LSM 880 upright confocal, or a Zeiss LSM 800.
+  Fluorescence:
+    name: Fluorescence
+    roi_response_series:
+      - name: NeuronFluorescence
+        description: Segmented regions of interest (ROIs) and baseline activity for the neuron indicator type.
+      - name: GliaFluorescence
+        description: Segmented regions of interest (ROIs) and baseline activity for the glial indicator type.
+  DfOverF:
+    name: DfOverF
+    roi_response_series:
+      - name: NeuronDfOverF
+        description: Segmented regions of interest (ROIs) and detrended activity for the neuron indicator type.
+      - name: GliaDfOverF
+        description: Segmented regions of interest (ROIs) and detrended activity for the glial indicator type.
+  ImageSegmentation:
+    plane_segmentations:
+      - name: NeuronVolumeSegmentation
+        description: Segmented ROIs for the neural cells.
+      - name: GliaVolumeSegmentation
+        description: Segmented ROIs for the glia cells.
+  ImagingPlane:
+    - name: NeuronVolumeSegmentation
+      description: Whole-brain optical imaging for the neuron filter.
+      device: Microscope
+      indicator: GCaMP6f
+      excitation_lambda: 488 nm
+      location: whole-brain
+      reference_frame: top-down
+      optical_channel:
+        - name: NeuronOpticalChannel
+          description: Detection filter (Semrock).
+          emission_lambda: 525-550 nm
+    - name: GliaVolumeSegmentation
+      description: Whole-brain optical imaging for the glial filter.
+      indicator: GCaMP6f
+      # Some single-color glia subjects used the jRGECO1b indicator
+      # For jRGECO1b indicator single-color, excitation is 561 nm
+      # with a 630-692 nm filter
```

Comment:
Need to check with Yu Mu to see if he has a mapping from subject IDs to indicators; the methods simply indicate that some of the subjects used in some of the panels of Fig. 2 used one or the other